gsa/src/gmp/commands/users.js

```diff
@@ -45,6 +45,8 @@ import {parseInt} from '../parser';
 import EntitiesCommand from './entities';
 import EntityCommand from './entity';
 
+import {COMPOSER_CONTENT_DEFAULTS} from 'web/components/dialog/composercontent';
```

Comment:
Sorry the gmp part must not depend on the web part. You could define the defaults here and import them in the web part or you only return an empty object and set the defaults. The only aim is to ensure not to crash if an error did occur.